### ISSUE TITLE
Certificate chain stored?

### ISSUE THREAD

**Issue by bhoeferlin** [valb3r/letsencrypt-helper#8]
The issued certificate works great when using a browser, but it seems that some intermediate certificates are missing, this the certificate chain is not stored in the keystore. Could this be? I'm not that deeply familiar with the topic...
When checking against openssl, I retrieve these issues: 
```
openssl s_client -connect www.example.com:443
CONNECTED(00000005)
depth=0 CN = www.example.com
verify error:num=20:unable to get local issuer certificate
verify return:1
depth=0 CN = www.example.com
verify error:num=21:unable to verify the first certificate
verify return:1
---
Certificate chain
 0 s:CN = www.example.com
   i:C = US, O = Let's Encrypt, CN = R3
```

**Comment by valb3r** [comment]
No, currently chain is not stored, only domain certificate, so client is required to build certificate chain themselves (since most of clients do have this chains themselves typically entire chain is not necessary). Adding this as a feature to store certificate chain

**Comment by valb3r** [comment]
@bhoeferlin Available in `0.2.3`

**Comment by bhoeferlin** [comment]
Works perfect - thanks!
